content/news/XXXX-XX-XX-gh-issue-migration-downtime.md

```diff
@@ -0,0 +1,21 @@
+title: Bugtracker temporarily unavailable
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
+publish a more detailed blogpost after the migration is done. For now, We'd
```

Comment:
```suggestion
For now, We'd
```
The sentence is repeated below.

Reply:
I'm aware, but I wanted to mention the extra blogpost upfront. Afterwards I need it to refer to the time when its published.

Reply:
I'm any case the sentence is incomplete and should be fixed or removed.

Reply:
fixed